ext/opentelemetry-ext-prometheus/src/opentelemetry/ext/prometheus/__init__.py

```diff
@@ -152,33 +152,46 @@ def _translate_to_prometheus(self, metric_record: MetricRecord):
             metric_name = self._prefix + "_"
         metric_name += self._sanitize(metric_record.instrument.name)
 
+        description = getattr(metric_record.instrument, "description", "")
         if isinstance(metric_record.instrument, Counter):
             prometheus_metric = CounterMetricFamily(
-                name=metric_name,
-                documentation=metric_record.instrument.description,
-                labels=label_keys,
+                name=metric_name, documentation=description, labels=label_keys
             )
             prometheus_metric.add_metric(
                 labels=label_values, value=metric_record.aggregator.checkpoint
             )
         # TODO: Add support for histograms when supported in OT
         elif isinstance(metric_record.instrument, ValueRecorder):
-            prometheus_metric = UnknownMetricFamily(
-                name=metric_name,
-                documentation=metric_record.instrument.description,
-                labels=label_keys,
-            )
-            prometheus_metric.add_metric(
-                labels=label_values, value=metric_record.aggregator.checkpoint
-            )
+            value = metric_record.aggregator.checkpoint
+            if isinstance(
+                value, MinMaxSumCountAggregator._TYPE  # pylint: disable=W0212
+            ):
+                value = metric_record.aggregator.checkpoint
```

Comment:
Seems like value is defined twice? Line 165

Reply:
Fixed in https://github.com/open-telemetry/opentelemetry-python/pull/945/commits/bb6b8ffda8e51e30b7ccffc9177a74db17c6bdcb